src/components/protocol_handler/src/service_status_update_handler.cc

```diff
@@ -28,39 +28,42 @@ void ServiceStatusUpdateHandler::OnServiceUpdate(
     const protocol_handler::ServiceType service_type,
     ServiceStatus service_status) {
   using namespace hmi_apis;
+  typedef utils::Optional<Common_ServiceUpdateReason::eType>
```

Comment:
@mked-luxoft nice typedef is it possible to move it to some header and use in more places ?